### DISCUSSION TITLE
task02 / 로그인 모달창 구현

### DISCUSSION THREAD

**Discussion by Achates09** [codestates/Hidden_Track#86] [Announcements]
### ISSUE
- Group: `client`
- Type: `feature`
- Detail: 

### TODO
1. [x] Logo
2. [ ] 아이디(유효성 검사)
3. [ ] 비밀번호(유효성 검사)
4. [x] 회원가입 페이지 이동 버튼
5. [x] 로그인 버튼
6. [ ] 소셜로그인 버튼
7. [x] 모달창 닫기 버튼(X)
8. [ ] 로그인 상태유지(advanced)

**Comment by James940522** [comment]
확인 했습니다.

**Reply by jjub0217** [reply]
엇. 아래 comments에 달아주시면 감사하겠습니다:)

**Comment by jjub0217** [comment]
1. 아이디(유효성 검사) => 저희가 아이디를 이메일 형식으로 하기로 했다면 필요한 부분이만, 영어와 숫자로만 이루어진 걸로 아이디를 하기로 했기 때문에 해당 task 는 현재 저희 플젝의 로그인 모달창에는 굳이 필요없다는 의견이 나왔기에 pass 합니다. 
2. 비밀번호(유효성검사) => 아이디(유효성 검사) 부분과 비슷한 이유로, 굳이 필요없다는 의견이 나왔기에 pass 합니다.
3. 로그인 상태유지(advanced) => 현재 시점으로는 Bareminimum 단계이므로, advanced 단계때 하기로 한 해당 task 는 task02번에서는 pass 합니다.
